include/capwap_config.hrl

```diff
@@ -97,3 +97,47 @@
 	  group_rekey,
 	  strict_group_rekey
 	 }).
+
+-define(DEFAULT_COMMON,
```

Comment:
this is used only once, why does it have to be in a header file?

Reply:
it's just predefined list for defaults. I think that it easier to find